### PR TITLE
admin tools in login menu, closes #599

### LINKED ISSUE #599: add admin tools to the Login menu
(@jywarren reformatted this on @ebarry's behalf):

Hi, this is a `first-timers-only` issue. This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem

An admin-only interface for searching for peoples' email addresses needs to be readily available; the URL is:

https://publiclab.org/useremail (though you won't be able to see it on the live site if you're not an admin!)

But it's hard to find or remember.

## Solution

We could put it in the Login menu in the upper right, in this template:

https://github.com/publiclab/plots2/blob/master/app/views/layouts/_header.html.erb#L175

Perhaps below a divider, and surrounded by `if` tags so it's only shown to admins:

```
<% if current_user.role == 'admin' %>
  <li class="divider"></li>
  <li><a href="/useremail">Email search</a></li> 
<% end %>
```

## Steps to Fix

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/developers) if you have questions!

### PR DESCRIPTION
**Pull request by 500swapnil**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] all tests pass -- `rake test:all`
* [v ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [v] pull request is descriptively named with #number reference back to original issue
* [v] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [ ] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Oh i see you did it here -- great! Added one more comment/request, thank you!

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -173,6 +173,10 @@
               <li><a href="/subscriptions"><%= t('layout._header.subscriptions') %></a></li>
               <li><a href="/profile/<%= current_user.username %>/likes"><%= t('layout._header.notes_liked') %></a></li> 
               <li class="divider"></li>
+	      <% if current_user.role == 'admin' %>
+	        <li class="divider"></li>
+		  <li><a href="/useremail">Email search</a></li>
```
Comment (jywarren) [inline app/views/layouts/_header.html.erb]: Yep! Perfect. Let's let the tests run and I'll pull it in. Thanks!

**Comment by jywarren** [comment]
Hi -- this looks perfect; merged! If you're looking for a new challenge, please take a look at our `help-wanted` list: https://github.com/publiclab/plots2/labels/help-wanted

Thanks! 🎈

**Comment by 500swapnil** [comment]
Thank you ! :)  Yeah I will have a look at it.

**Pull request merged**